lms/djangoapps/course_home_api/dates/v1/views.py

```diff
@@ -85,6 +86,7 @@ def get(self, request, *args, **kwargs):
         user_timezone = user_timezone_locale['user_timezone']
 
         data = {
+            'is_archived': course.end < datetime.now(),
```

Comment:
It'd be nice if this were an `@property` or something.

Reply:
Turns out, it already existed, I just had to use it. (`has_ended()` was the correct method to call)